plugins/module_utils/gitlab.py

```diff
@@ -7,23 +7,23 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-import json
 from distutils.version import StrictVersion
 
 from ansible.module_utils.basic import missing_required_lib
-from ansible.module_utils.urls import fetch_url
 from ansible.module_utils.common.text.converters import to_native
 
 try:
     from urllib import quote_plus  # Python 2.X
+    from urlparse import urljoin
 except ImportError:
-    from urllib.parse import quote_plus  # Python 3+
+    from urllib.parse import quote_plus, urljoin  # Python 3+
 
 import traceback
 
 GITLAB_IMP_ERR = None
 try:
     import gitlab
+    import requests
```

Comment:
This add a dependency to the module, that might not even be required if the user is authenticating with `api_token`. 

At the very least, this new dependency should show up in the modules' documentation.

Reply:
requests is a dependency of gitlab's python api so I thought it was OK to use it. Do you still want me to explicitly list it?